codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -215,7 +215,7 @@ private void activateInternal(DefaultStream stream) {
     private void deactivateInternal(DefaultStream stream) {
         if (activeStreams.remove(stream)) {
```

Comment:
I just double checked with the HTTP/2 spec and it sounds like our notion of "active" is exactly what should count towards `SETTINGS_MAX_CONCURRENT_STREAMS`.  Since the spec doesn't really provide a term for streams in this state (i.e. open or half-open), I think "active" is reasonable.  So I've changed everything over to use the "active" terminology.  I also cleaned up a few other things related to the Http2Connection API as well.